Source/WebKit/UIProcess/API/glib/WebKitFormClient.cpp

```diff
@@ -37,7 +37,7 @@ class FormClient final : public API::FormClient {
     }
 
 private:
-    void willSubmitForm(WebPageProxy&, WebFrameProxy&, WebFrameProxy&, WebKit::FrameInfoData&&, WebKit::FrameInfoData&&, const Vector<std::pair<String, String>>& values, API::Object*, CompletionHandler<void()>&& completionHandler) override
+    void willSubmitForm(WebPageProxy&, WebFrameProxy&, WebFrameProxy&, WebKit::FrameInfoData&&, WebKit::FrameInfoData&&, const Vector<std::pair<String, String>>& values, API::Object*, const WTF::URL&& requestURL, const WTF::String&& method, CompletionHandler<void()>&& completionHandler) override
```

Comment:
WebKit layer does not use -Wunused-parameter to avoid the need for using `UNUSED_PARAM()` all over the place, which is why I expect this to survive EWS. But it's still nicer to not name parameters that are not used.

Reply:
I could go either way here, since URL and String are very generic types.